wiremock/responses/questionGroups.json

```diff
@@ -1297,7 +1297,7 @@
   "ROSH": {
     "type": "group",
     "groupId": "65a3924c-4130-4140-b7f4-cc39a52603bb",
-    "groupCode": "pre_sentence_assessment",
+    "groupCode": "ROSH",
```

Comment:
actually this one is pre_sentence_assessment

Reply:
the mocks are the same

Reply:
👍